src/pages/console/adminui/access-management.mdx

```diff
@@ -35,7 +35,9 @@ Follow these instructions to add and manage team members and their access to a p
   * Choose **Delete**. In the **Delete users** window, confirm the delete action.
 
 <Callout warning>
-If a team member logs into Amplify Studio, their login token is valid for 60 minutes, unless explicitly logged out. When you change a team member's permission from *full access* to *manage-only* or when you delete a team member's access, the team member can continue accessing Amplify Studio with their previously granted permissions until their token expires.
+
+If a team member logs into Amplify Studio, their login token is valid for 60 minutes, unless they explicitly log out. When you change a team member's permission from **Full access** to **Manage only** or when you delete a team member's access, the team member can continue accessing Amplify Studio with their previously granted permissions until their token expires.
```

Comment:
Is this correct? I almost never have to sign back into Studio. I thought Studio access was unlimited via Amplify Auth.

Reply:
fixed this is 5hrs 30 minutes now